PowerUp/app/src/main/res/layout/activity_main.xml

```diff
@@ -29,22 +29,28 @@
         android:layout_height="@dimen/powerup_icon_height"
         android:src="@drawable/ic_launcher" />
 
-    <ImageButton
+    <Button
         android:id="@+id/startButtonMain"
         android:layout_width="@dimen/start_button_width"
         android:layout_height="@dimen/start_button_height"
-        android:background="@drawable/buttons" />
+        android:background="@drawable/brown_button_background"
+        android:text="@string/start"
```

Comment:
No resource found that matches the given name (at 'text' with value '@string/start').